iOS/ThirdPartyAuth/README.md

```diff
@@ -16,16 +16,16 @@ If you want to run the sample, you will need to set up an auth0 account and conf
 1. Follow the instructions in the [iOS Readme](../README.md).
 1. Create an client on [auth0](https://auth0.com/).
```

Comment:
As with the Android README.md, the common auth0 instructions should be moved into an AUTH0.md file that is referenced from here.